shell/agents/Microsoft.Azure.Agent/ChatSession.cs

```diff
@@ -231,6 +241,9 @@ private HttpRequestMessage PrepareForChat(string input)
         var request = new HttpRequestMessage(HttpMethod.Post, _conversationUrl) { Content = content };
 
         request.Headers.Authorization = new AuthenticationHeaderValue("Bearer", _token);
+        // These headers are for server side telemetry. We refresh correlation ID for each query.
+        request.Headers.Add("CorrelationId", NewCorrelationID());
+        request.Headers.Add("ClientType", "AIShell");
```

Comment:
These two headers do make sense for server side telemetry, so it will be good if they can be passed to the endpoint. I am asking portal team about whether these could be blocked by Orchesrator.

Reply:
Solution:

- `CorrelationId` can be deprecated. We will use `conversationId` and `ReplyToId` from Orchestrator to correlate between 3 Telemetry. 
- `ClientType` should be left here, portal team will try to pass it thourgh Orchestrator so that the Endpoint can know where the request comes from.